src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -2531,14 +2448,7 @@ protected override object GetItemDynamicParameters(string path)
         /// </returns>
         protected override object GetChildItemsDynamicParameters(string path, bool recurse)
         {
-            if (DownLevelHelper.NativeFilteringSupported())
-            {
-                return new CertificateProviderDynamicParameters();
-            }
-            else
-            {
-                return new CertificateProviderCodeSigningDynamicParameters();
-            }
+            return new CertificateProviderCodeSigningDynamicParameters();
```

Comment:
This is the code path that is always taken on `Windows PowerShell`.  It should work.